tests/integration/api_v3_product_read.t

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/perl -w
+
+use ProductOpener::PerlStandards;
+
+use Test::More;
+use ProductOpener::APITest qw/:all/;
+use ProductOpener::Test qw/:all/;
+use ProductOpener::TestDefaults qw/:all/;
+
+use File::Basename "dirname";
+
+use Storable qw(dclone);
+
+remove_all_users();
+
+remove_all_products();
+
+wait_dynamic_front();
```

Comment:
FYI, in [my pending PR](https://github.com/openfoodfacts/openfoodfacts-server/pull/7453), I added a `wait_application_ready` that also wait for server to be ready, for in some case, test where failing because apache server was not ready to serve request.

Reply:
Perfect.